SwedbankPaySDK/Classes/SwedbankPaySDKController.swift

```diff
@@ -199,12 +231,29 @@ public final class SwedbankPaySDKController: UIViewController {
     }
     
     /// Creates paymentOrder
-    private func createPaymentOrder(_ backendUrl: URL) {
-        viewModel.createPaymentOrder(backendUrl, successCallback: { [weak self] operationsList in
-            self?.createPaymentOrderURL(operationsList)
-        }, errorCallback: { [weak self] problem in
-            self?.paymentFailed(failureReason: .Problem(problem))
-        })
+    private func createPaymentOrder() {
+        viewModel.createPaymentOrder { [weak self] in
+            self?.handleCreatePaymentOrderResult(result: $0)
+        }
+    }
+    
+    private func handleResult<T>(result: Result<T, Error>, onSuccess: (T) -> Void) {
+        switch result {
+        case .success(let t):
```

Comment:
Codacy found an issue: [Variable name should be between 3 and 40 characters long: 't'](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-ios/pullRequest?prid=6260585)